.gitignore

```diff
@@ -8,6 +8,9 @@ bundles/io/org.openhab.io.multimedia.tts.marytts/lib/
 */plugin.xml_gen
 targetplatform/*.launch
 
+.idea
+**/*.iml
```

Comment:
@kaikreuzer wdyt? Is it maybe time to support the Intellij users by adding these lines to the .gitignore?

Reply:
Fine for me.